File: lib/fangraphs.js
const r = require('r-wrapper').async;

exports.batterGameLogs = async (fangraphsid, season) => {
    const res = await r('./R/fangraphs.R', 'batter_game_logs', {
        fangraphsid: fangraphsid,
        season: season
    });
    
    return res;
}

File: R/fangraphs.R
batter_game_logs <- function(fangraphsid, season=2021) {
  x <- baseballr::batter_game_logs_fg(fangraphsid, season);
  return(x)
}

pitcher_game_logs <- function(fangraphsid, season=2021) {
  x <- baseballr::pitcher_game_logs_fg(fangraphsid, season);
  return(x)
}